anxcloud/resource_virtual_server.go

```diff
@@ -92,6 +92,12 @@ The virtual_server resource allows you to configure and run virtual machines.
 							log.Fatalf("[ERROR] unable to force new '%s': %v", key, err)
 						}
 					}
+					if newNet.BandwidthLimit != oldNets[i].BandwidthLimit {
+						key := fmt.Sprintf("network.%d.bandwidth_limit", i)
+						if err := d.ForceNew(key); err != nil {
+							log.Fatalf("[ERROR] unable to force new '%s': %v", key, err)
+						}
```

Comment:
According to SYSENG-1806, one acceptance criteria is:

> Changing the speed updates the resource in-place and doesn't replace the existing VM.

This would go against said criteria.

Reply:
Oopsie! Sorry, should be address now!

Reply:
As discussed, this change forces a replacement and refactoring will be done once we refactor the virtual server resource